react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+let counter = 0;
+
+class HotButton extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { isClicked: false };
```

Comment:
Here you only need to keep track of the clicks in state. So you would do something like:

```JSX
this.state = {
  counter: 0
};
```